doc/airline.txt

```diff
@@ -888,6 +888,12 @@ lsp <https://github.com/prabirshrestha/vim-lsp>
 <
 * lsp close_lnum_symbol >
   let airline#extensions#lsp#close_lnum_symbol = ')'
+<
+* lsp progress skip time
+  Suppresses the frequency of status line updates.
```

Comment:
what should it be set to disable? `0`

Reply:
Yes.
Set 0 to disable.
I've added this to `doc/airline.txt`